robottelo/ui/contentviews.py

```diff
@@ -707,8 +707,12 @@ def fetch_version_packages(self, name, version_name):
         """Return a list of all the packages inside specific content view
         version"""
         self.click(self.version_search(name, version_name))
-        self.click(tab_locators['contentviews.tab_version_packages'])
+        no_packages_tab = self.wait_until_element_is_not_visible(
```

Comment:
I prefer to use names like `packages_tab_is_not_visible` and `errata_tab_is_not_visible`, but if string get to more than 80 symbols then change is not worth it
What do you think?
It is just a nitpick and I can merge PR right away, just let me know